projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -639,6 +659,55 @@ public function edit_dsp_generic( $path, $req, $args = array() ) {
 		);
 	}
 
+	/**
+	 * Will check the posts for prices and add them to the posts array
+	 *
+	 * @param WP_REST_Request $posts The posts object.
+	 * @return array|WP_Error
+	 */
+	protected function add_prices_in_posts( $posts ) {
+
+		if ( ! function_exists( 'wc_get_product' ) ||
+			! function_exists( 'wc_get_price_decimal_separator' ) ||
+			! function_exists( 'wc_get_price_thousand_separator' ) ||
+			! function_exists( 'wc_get_price_decimals' ) ||
+			! function_exists( 'get_woocommerce_price_format' ) ||
+			! function_exists( 'get_woocommerce_currency_symbol' )
+		) {
+			return $posts;
+		}
+
+		foreach ( $posts as $key => $post ) {
```

Comment:
As a general advice, avoid using `$post` as it's "reserved" by `global $post` and can cause confusion.